source/backups.rst

```diff
@@ -1,3 +1,5 @@
+.. _backups:
+
 Providing Backups
 =================
```

Comment:
I think it is needed to explain in `Making on-demand backup` section, what exactly needed to put into `pxcCluster` and `storageName` fields